src/main/java/com/bitquest/bitquest/BitQuest.java

```diff
@@ -811,6 +811,8 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
                     } catch (IOException e1) {
                         e1.printStackTrace();
                     }
+                    if (sendAmount >= 20000)
+                    {
                     try {
```

Comment:
If you haven't run, tested, and debugged the code yourself, I would not submit a PR for it until you have.